.devcontainer/devcontainer.json

```
@@ -0,0 +1,5 @@
+{
+  "image": "aziotbld/embedded-codespace",
```

Can we generalize this and make it for all of Embedded C
```suggestion
  "image": "azsdk/embeddedc-iot-codespace",
```

---

this is mainly because we have a Docker account we use for building IoT tests and things. I stuck it there since I have the credentials for that. If we wanted to create our own account or something we could.